interface/forms/LBF/printable.php

```diff
@@ -30,6 +33,10 @@
 if ($patientid < 0) {
     $patientid = 0 + $pid; // -1 means current pid
 }
+// PDF header information
+$patientname = getPatientName($patientid);
+$patientdob = getPatientData($patientid, $given = "DOB");
```

Comment:
Just noted this bug in the second parameter above; the current call will likely break:
```
$patientdob = getPatientData($patientid, "DOB");
```
After you fix this and you test that this code works, then let me know, and I'll bring this PR into codebase. thanks! -brady

Reply:
Yes getPatientData($patientid, $given = "DOB"); works.

Reply:
all you're doing is setting an unused variable which in the end gets parsed and 'DOB' is passed. still, why?

Reply:
Is this question for me?

Reply:
Not a question. Just pointing out the param $given=anything is pointless. I consider it bad practice to call functions or resolve value for a param inside a function call param args. But, i'm old school from old school languages.

Reply:
to sum this thread up, note @juggernautsei changed it to:
```
$patientdob = getPatientData($patientid, "DOB");
```